lib/graphql/tracing/skylight_tracing.rb

```diff
@@ -18,6 +18,12 @@ def platform_trace(platform_key, key, data)
         if (query = data[:query])
           title = query.selected_operation_name || "<anonymous>"
           category = platform_key
+          # Assign the endpoint so that queries will be grouped
+          current_instance = Skylight::Instrumenter.instance
```

Comment:
This doesn't work with the latest version of the skylight gem (2.0.2). Instead, do something like this:

```
inst = Skylight.instrumenter
current_trace = inst ? inst.current_trace : nil
if current_trace
  endpoint = "GraphQL/#{query.operation_type}.#{title}"
  current_trace.endpoint = endpoint
end
```